features/step_definitions/money_manager_steps.rb

```diff
@@ -0,0 +1,22 @@
+Given(/^I am on the Money Manager tool$/) do
```

Comment:
@picandocodigo it looks great! The name for this feature could be changed to reflect the primary feature which is _Enhance sign-out behaviour_.